relayer/pkg/chainlink/keys/key.go

```diff
@@ -11,101 +11,101 @@ import (
 )
 
 // Raw represents the Stark private key
-type StarkRaw []byte
+type Raw []byte
 
 // Key gets the Key
-func (raw StarkRaw) Key() StarkKey {
+func (raw Raw) Key() Key {
 	privKey := starksig.PrivateKey{}
 	privKey.D = new(big.Int).SetBytes(raw)
 	privKey.PublicKey.Curve = curve
 	privKey.PublicKey.X, privKey.PublicKey.Y = curve.ScalarBaseMult(raw)
 
-	return StarkKey{
+	return Key{
 		privkey: privKey,
 	}
 }
 
 // String returns description
-func (raw StarkRaw) String() string {
+func (raw Raw) String() string {
 	return "<StarkNet Raw Private Key>"
 }
 
 // GoString wraps String()
-func (raw StarkRaw) GoString() string {
+func (raw Raw) GoString() string {
 	return raw.String()
 }
 
-var _ fmt.GoStringer = &StarkKey{}
+var _ fmt.GoStringer = &Key{}
 
 // Key represents StarkNet key
-type StarkKey struct {
+type Key struct {
 	privkey starksig.PrivateKey
 }
 
 // New creates new Key
-func New() (StarkKey, error) {
+func New() (Key, error) {
 	return newFrom(crypto_rand.Reader)
 }
 
 // MustNewInsecure return Key if no error
-func MustNewInsecure(reader io.Reader) StarkKey {
+func MustNewInsecure(reader io.Reader) Key {
 	key, err := newFrom(reader)
 	if err != nil {
 		panic(err)
 	}
 	return key
 }
 
-func newFrom(reader io.Reader) (StarkKey, error) {
+func newFrom(reader io.Reader) (Key, error) {
 	privKey, err := starksig.GenerateKey(curve, reader)
 	if err != nil {
-		return StarkKey{}, err
+		return Key{}, err
 	}
-	return StarkKey{
+	return Key{
 		privkey: *privKey,
 	}, nil
 }
 
 // ID gets Key ID
-func (key StarkKey) ID() string {
-	return key.PublicKeyStr()
+func (key Key) ID() string {
+	return key.ContractAddressStr()
 }
 
-// Not actually public key, this is the derived contract address
+// this is the derived contract address, the contract is deployed using the StarkKeyStr
 // This is the primary identifier for onchain interactions
 // the private key is identified by this
-func (key StarkKey) PublicKeyStr() string {
+func (key Key) ContractAddressStr() string {
```

Comment:
Let's call this an account:
```suggestion
func (key Key) AccountAddressStr() string {
```